step-templates/redgate-create-database-release.json

```diff
@@ -131,16 +131,25 @@
                                                }
                        },
                        {
-                           "Name":  "SpecificModuleVersion",
+                           "Name":  "DLMAutomationSpecificModuleVersion",
```

Comment:
I understand the desire to want to change this variable name to be consistent, but anyone using this step will be forced to update this value (if they have it set).  Or, they could lose the value they have set already.  Is it absolutely required to change this variable name?

![image](https://user-images.githubusercontent.com/38642825/85149211-a2e28400-b216-11ea-8d92-cbe7bb4ca026.png)

Reply:
I changed it to match the requirement in step 4 of your checklist, but I can change this back to the previous value if you'd prefer.

Reply:
It is a risk/reward concern.  I appreciate you following the guidelines.  However, that parameter was added two years ago and we haven't gotten a complaint about it (unless you have).  I think it is safe to revert back to the previous value in this case.

Reply:
Great, thank you! I'll do that, and update this pull-request when I've done so.